airflow-core/src/airflow/jobs/triggerer_job_runner.py

```diff
@@ -463,6 +468,9 @@ def emit_metrics(self):
         )
 
     def _send(self, msg: BaseModel):
+        self.trigger_stdin.write(msg.model_dump_json().encode("utf-8") + b"\n")  # type: ignore[union-attr]
```

Comment:
That type ignore seems of. It wasn't necessary before and it seems really similar now: